packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -223,12 +223,38 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
       waitForDOMAndCameraDetails: {
         after: {
           0: {
-            target: 'start',
+            target: 'initializeLivenessStream',
             cond: 'hasDOMAndCameraDetails',
           },
           10: { target: 'waitForDOMAndCameraDetails' },
         },
       },
+      initializeLivenessStream: {
+        invoke: {
+          src: 'openLivenessStreamConnection',
+          onDone: {
+            target: 'notRecording',
+            actions: [
+              'updateLivenessStreamProvider',
+              'spawnResponseStreamActor',
+            ],
+          },
+        },
+      },
+      notRecording: {
```

Comment:
Would it make sense to call this "connecting"? That seems to be what it's called in the e2e test?

Reply:
Renamed it already!